pkg/cluster/describe.go

```diff
@@ -78,6 +78,19 @@ func PrintClusterDesctipion(connection *sdk.Connection, cluster *cmv1.Cluster) e
 		creator = "N/A"
 	}
 
+	// Find the details of the shard
+	shardPath, err := connection.ClustersMgmt().V1().Clusters().
+		Cluster(cluster.ID()).
+		ProvisionShard().
+		Get().
+		Send()
+	var shard string
+	if shardPath == nil || err != nil {
+		shard = "N/A"
```

Comment:
This exposes knowledge about shards to users without authorization. I would still leave as a conditional print (only print it out if it's not `nil`)

Reply:
:+1:

Reply:
Done